src/config.h

```diff
@@ -358,6 +358,25 @@ void setcpuaffinity(const char *cpulist);
 #define ATTRIBUTE_TARGET_AVX2
 #endif
 
+#if defined (HAVE_AVX2)
+#define ATTRIBUTE_TARGET_AVX2_POPCOUNT __attribute__((target("avx2,popcnt")))
+#else
+#define ATTRIBUTE_TARGET_AVX2_POPCOUNT
+#endif
```

Comment:
what about making it simpler.
```suggestion
/* Check if we can compile AVX2 code */
#if defined (__x86_64__) && ((defined(__GNUC__) && __GNUC__ >= 5) || (defined(__clang__) && __clang_major__ >= 4))
#if defined(__has_attribute) && __has_attribute(target)
#define HAVE_AVX2
#define ATTRIBUTE_TARGET_AVX2 __attribute__((target("avx2")))
#define ATTRIBUTE_TARGET_AVX2_POPCOUNT __attribute__((target("avx2,popcnt")))
#endif
#endif
```

Reply:
addressed in https://github.com/redis/redis/pull/14309/commits/0481b176d50b9aa0eb5505a76431f635aa634daa